contents/guides/users/aspects.md

```diff
@@ -0,0 +1,41 @@
+---
+title: What are aspects and how do I use them?
+category: User guides
+---
+
+## What is an aspect?
+
+Think of your life and the people you know. Each person is a part of one or more aspects of your life. They might be a member of your family, a close friend, a work colleague or someone you play sport or music with or with whom you share a particular interest. They might be someone you just like sharing jokes with online. Or they might be part of more than one of these aspects of your life.
+
+diaspora\* works in exactly the same way: Aspects are the way you group your contacts on diaspora\*.
+
+You place contacts into one or more “aspects” of your life on diaspora\* based on which aspect(s) of your life they are part of. In this way, you control which of your contacts sees each post you make to diaspora\*, by posting to a particular aspect or aspects. This means that you can easily share the right things with the right people.
```

Comment:
"which aspect(s) of your life they are part of" – remove "of your life".
"particular aspect or aspects" –> "particular aspect or group of aspects" for clarity.

Reply:
yes, this makes it easier to read.